app/helpers/application_helper.rb

```diff
@@ -158,13 +158,17 @@ def translation(key, options = {})
     translated_string = t(key, options)
     options[:fallback] = false
     translated_string2 = t(key, options)
-
-    if current_user&.has_tag('translation-helper') && translated_string2.include?("translation missing") && !translated_string.include?("<")
-      raw(%(<span>#{translated_string} <a href="https://www.transifex.com/publiclab/publiclaborg/translate/#de/$?q=text%3A#{translated_string}">
-          <i data-toggle='tooltip' data-placement='top' title='Needs translation? Click to help translate this text.' style='position:relative; right:2px; color:#bbb; font-size: 15px;' class='fa fa-globe'></i></a>
-       </span>))
-    else
+     # To prevent no method error when app/models/concerns/node_shared.rb calls app/views/notes/_card.html.erb
+     # which then calls this method without current_user
+    begin
+      if current_user&.has_tag('translation-helper') && translated_string2.include?("translation missing") && !translated_string.include?("<")
+        raw(%(<span>#{translated_string} <a href="https://www.transifex.com/publiclab/publiclaborg/translate/#de/$?q=text%3A#{translated_string}">
+            <i data-toggle='tooltip' data-placement='top' title='Needs translation? Click to help translate this text.' style='position:relative; right:2px; color:#bbb; font-size: 15px;' class='fa fa-globe'></i></a>
+         </span>))
+      else
+        raw(translated_string)
+      end
+     rescue
       raw(translated_string)
-    end
+     end
   end
-end
```

Comment:
This might be what is missing and causing the tests to fail @Priyaraj17, you can consider adding it